<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-6TAQC2O" failures="1" tests="1" name="com.promilo.automation.signupandlogin.ValidLogin" time="11.429" errors="0" timestamp="2025-08-26T11:32:09 IST" skipped="0">
  <testcase classname="com.promilo.automation.signupandlogin.ValidLogin" name="validLogin" time="11.429">
    <failure message="Locator expected to be visible
Call log:
  - Assert &quot;isVisible&quot; with timeout 5000ms
  - waiting for locator(&quot;//img[@alt=&#039;my stuff&#039;]&quot;)" type="org.opentest4j.AssertionFailedError">
      <![CDATA[org.opentest4j.AssertionFailedError: Locator expected to be visible
Call log:
  - Assert "isVisible" with timeout 5000ms
  - waiting for locator("//img[@alt='my stuff']")
at com.microsoft.playwright.impl.AssertionsBase.expectImpl(AssertionsBase.java:69)
at com.microsoft.playwright.impl.AssertionsBase.expectImpl(AssertionsBase.java:50)
at com.microsoft.playwright.impl.LocatorAssertionsImpl.expectTrue(LocatorAssertionsImpl.java:463)
at com.microsoft.playwright.impl.LocatorAssertionsImpl.isVisible(LocatorAssertionsImpl.java:458)
at com.microsoft.playwright.assertions.LocatorAssertions.isVisible(LocatorAssertions.java:845)
at com.promilo.automation.signupandlogin.ValidLogin.validLogin(ValidLogin.java:98)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
    </failure>
  </testcase> <!-- validLogin -->
</testsuite> <!-- com.promilo.automation.signupandlogin.ValidLogin -->
